sdk/containerregistry/container-registry/review/container-registry.api.md

```diff
@@ -149,6 +149,14 @@ export enum KnownArtifactOperatingSystem {
     Windows = "windows"
 }
 
+// @public
+export enum KnownContainerRegistryAudience {
```

Comment:
Just out of curiosity - is the Known prefix needed?  (Would we also have an enum called UnknownContainerRegistryAudience :) )  I'm not super concerned about the language difference, but I would like to try to understand it better.

Reply:
We have "Known"-prefix for our extensible enums (e.g., KnownArtifactArchitecture) from codegen so I also use it here.

Reply:
Ah, cool.  TIL! :)